pkg/dbdump/dbdump.go

```diff
@@ -0,0 +1,238 @@
+func (c *Collector) exportDBDump(destDir string) error {
+	c.log.Println("Exporting db dump")
+
+	if options.DBType == "postgres" {
+		// In case of a postgres, the dump was created as a local file, so all is left to
+		// do is tar it
+		var cmd *exec.Cmd
+
+		// Compose the path of the tar output
+		tarPath := fmt.Sprintf("%s.tar.gz", c.dbDumpFolderPath)
+
+		// Genrate the tar command
+		if destDir != "" {
+			cmd = exec.Command("tar", "-C", destDir, "-cvzf", tarPath, c.folderName)
+			} else {
+			cmd = exec.Command("tar", "-cvzf", tarPath, c.folderName)
+		}
+
+		// Execute the tar command
+		if err := cmd.Run(); err != nil {
+			c.log.Printf(`❌ failed to tar db dump`)
+			return err
+		}
+	} else {
+		// In case of a mongo db, the db dump was generated and tar'd at the remote machine,
+		// all is left to do is copy the tarball to the local machine
+		var localPath string
+
+		// Compose the local path to export the tarball
+		if destDir != "" {
+			localPath = fmt.Sprintf("%s/%s%s", destDir, c.folderName, ".tar.gz")
+		} else {
+			localPath = fmt.Sprintf("%s%s", c.folderName, ".tar.gz")
+		}
+
+		// Compose the full path to copy the tarball from
+		fullTarPath := fmt.Sprintf("%s:%s", "noobaa-db-0", c.remoteTarPath)
+		
+		// Create the command to copy the tarball
+		cmd := exec.Command("kubectl", "cp", fullTarPath, localPath)
+		// handle custom path for kubeconfig file,
+		// see --kubeconfig cli options
+		if len(c.kubeconfig) > 0 {
+			cmd.Env = append(cmd.Env, "KUBECONFIG=" + c.kubeconfig)
+		}
+
+		// Execute the command, copying the tarball
+		if err := cmd.Run(); err != nil {
+			c.log.Printf(`❌ failed to copy remote tar file`)
+			return err
+		}
+	}
+
+	return nil
+}
+
+// Remove raw resources from local or remote machines
+func (c *Collector) deleteRawResources() {
+	if options.DBType == "postgres" {
+		// In case of postgres, the db dump was created locall, so all is left
+		// to do is remove the raw dump
+
+		// Compose the deletion command
+		cmd := exec.Command("rm", "-rf", c.dbDumpFolderPath)
+
+		// Execute the deletion of raw resources
+		if err := cmd.Run(); err != nil {
+			c.log.Printf(`❌ failed to tar remove dump folder`)
+		}
+	} else {
+		// In case of mongo db, the dumo was created and tar'd at the remote machine,
+		// all is left to do is clean the remote machine
+
+		// Compose the deletion command of the raw resources
+		cmd := exec.Command("kubectl", "exec", "-it", "pod/noobaa-db-0", "--", "rm", "-rf", c.remoteFolderPath)
+
+		// Execute deletion of the raw resources
+		if err := cmd.Run(); err != nil {
+			c.log.Printf(`❌ failed to remove remote dump folder`)
+		}
+
+		// Compose the deletion command of the tar'd resources
+		cmd = exec.Command("kubectl", "exec", "-it", "pod/noobaa-db-0", "--", "rm", c.remoteTarPath)
+
+		// Execute deletion of the tar'd resources	
+		if err := cmd.Run(); err != nil {
+			c.log.Printf(`❌ failed to tar remove dump tar file`)
+		}
+	}
+}
```

Comment:
Add new line

Reply:
Fixed.
This should have been caught by lint check though.